### PR TITLE
Create Radon Sensor Signoff.md

### PR DESCRIPTION
**Pull request by JoshuaEgwuatu**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
Put the constraints into a numbered table for easier reading. These shouldn't be in paragraph format. Instead the constraint should be directly stated.

**Review by JesseTNRoberts — COMMENTED** [review]
This doesn't meet the expectations for a signoff document. There is a required set of sections that have to be completed and a specific structure.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Radon Sensor Signoff
+
+# Function of the subsystem
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md:3]: Be brief.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Size and Weight is also a consideration. The radon sensor (RD200) needs to be compact and lightweight so that it can easily be integrated into the robot. This allows the robot to maintain its mobility and functionality, even in small crawl space areas. The RD200 sensor with its dimensions of 80mm x 120mm and a weight of 240g should be accommodated.
+
+
+The Integration Complexity, while connecting the Radon Eye RD200 is relatively simple, integrating the sensor into the robot's data collection and processing system may pose challenges. It is important to carefully consider the robot's architecture and ensure compatibility with the sensor. Additionally, the sensor should be able to communicate with the microcontroller using protocols like Serial Peripheral Interface (SPI) and Inter-Integrated Circuit (I2C).
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md]: This seems true. But we need to know what the constraints are regarding integrating the sensor.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
+# Radon Sensor Signoff
+
+# Function of the subsystem
+
+The purpose of the radon sensor in the Autonomous Crawl Space Inspection Robot is to identify and measure the levels of radon gas in the environment. Radon is a hazardous radioactive gas that is commonly found in crawl spaces, resulting from the decay of uranium in natural stone. By integrating a radon sensor into the robot, it can gather real-time information on the radon levels present in the crawlspace. The sensor provides readings that indicate the concentration of radon gas, enabling early detection of elevated levels. By including a radon sensor, the Autonomous Crawlspace Inspection Robot improves its ability to collect environmental data, offering valuable insights into potential health risks and enabling proactive measures to protect against the dangers of radon exposure.
+
+The Radon Eye RD200 was selected by the team due to its compact and lightweight nature, distinguishing it from other radon sensors. It maintains dependable precision and accuracy when detecting radon gas levels. This guarantees the acquisition of reliable data for evaluating potential health hazards.
+
+# Constraints:
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md]: Number the constraints in the table, that way when you analyze them, it is easy to refer to the number.

**Review by JesseTNRoberts — COMMENTED** [review]
The design isn't buildable. It doesn't connect to anything?

File: Environmental Sensing/Radon Sensor Signoff.md (hunk 1 above)
Comment (JesseTNRoberts) [reply Environmental Sensing/Radon Sensor Signoff.md:3]: The purpose of this section is to state the function. Don't talk about components. Don't talk about Radon in general. 

State clearly what this subsystem is to do and nothing more.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+## Constraint 2:
+
+The sensor has a UART interface that will be used to communicate with the Arduino Nano. This will allow for data to be easier to track. The electrical schematic will show how these connections will take place.
+
+![Capture](https://github.com/JoshuaEgwuatu/Fall-2023-Autonomous-Crawlspace-Inspection-Robot/assets/110966922/8c94b023-226e-47b6-a670-f79a3ce60f19)
+An example of how we will communicate with the sensor
+
+## Constraint 3:
+
+The sensor accuracy meets the accuracy requirement of 0.5pCi/L or less. The sensor has been proven to have at least 90% accuracy according to Radon FTLab.
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md]: 90% of what? There's a bit of ambiguity in this.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Constraint 4:
+
+The sensor is powered by corded electricity with a voltage input of 12 volts and an amperage of 0.1 amps. The group has decided to use another battery pack to power this sensor separately from the rest of the sensors to not take away from the overall battery life of the robot.
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md]: Not acceptable to add another battery.

File: Environmental Sensing/Radon Sensor Signoff.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+## Constraint 5:
```
Comment (JesseTNRoberts) [inline Environmental Sensing/Radon Sensor Signoff.md]: I'm not certain what the constraint means. And what does it mean to display in intervals of 10 mins? How long does it take to get a measurement? How fast should the measurements be obtainable?

**Review by JesseTNRoberts — COMMENTED** [review]
Signoff document is not in the appropriate order. 

How is this supposed to work with the system sense it doesn't connect? 

Further, this does not meet the standard set by the previous team regarding how sensors should interface does it?

**Review by JesseTNRoberts — COMMENTED** [review]
The functional statement and the constraints here are very good! My question is, how often is a reading needed?

Further, a pcb is mentioned but I don't see one here.

Also, it would be best to have the circuit be part of the larger robot circuit schematic.

**Review by JesseTNRoberts — COMMENTED** [review]
Is the radon sensor serial communication compatible with the Arduino? Be sure to address in analysis.

**Review by JesseTNRoberts — COMMENTED** [review]
I'm not clear on how you are certain constraint 5 is met. It seems to give a 60 minute moving average. And I'm not sure that's what we want, is it?

Or at least, it isn't what you have proposed.

**Review by JesseTNRoberts — COMMENTED** [review]
Saying that it can communicate is not the same as showing that it can. You need to show that this is the case beyond only claiming it.

**Review by JesseTNRoberts — APPROVED** [review]
Is there a reason to not use a cheaper Arduino? I'm approving but I want to use a cheaper Arduino.

**Comment by JoshuaEgwuatu** [comment]
Would this version of the Arduino Nano suffice?
https://www.amazon.com/Deegoo-ATmega328P-Microcontroller-Board-Arduino/dp/B07R9VWD39/ref=asc_df_B07R9VWD39/?tag=hyprod-20&linkCode=df0&hvadid=459728334703&hvpos=&hvnetw=g&hvrand=10288603638586992834&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1025954&hvtargid=pla-945070251730&psc=1

**Pull request merged**